### PR TITLE
Seeders added

### PR DESCRIPTION
**Pull request by gregpmillr**
Added classes for seeders, generating on app startup, also calls a display method to demonstrate how queries will work.

### CODE REVIEW

**Comment by gregpmillr** [comment]
See issue #12

**Comment by ericdesj** [comment]
Nice, does the DB 'migrate' now?

**Comment by ericdesj** [comment]
Rather, does it drop the table upon load?

**Comment by gregpmillr** [comment]
@ericdesj No I'm still looking into this. Android handles migrations itself when the db is updated rather than when it's loaded, mainly because on the main thread there's already so much happening, it ramps up the loading time quite a bit. Leaving these links for reference to help fix the issue on my next PR:

http://stackoverflow.com/questions/15667965/why-we-need-to-onupgrade-method-in-sqliteopenhelper-class

http://stackoverflow.com/questions/21881992/when-is-sqliteopenhelper-oncreate-onupgrade-run

https://developer.android.com/reference/android/database/sqlite/SQLiteOpenHelper.html

**Comment by ericdesj** [comment]
right on, was just wondering

File: app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
+ */
+
+public class CourseSeeder {
+
+    public static void insert(DBHelper db){
+        db.course.insert("Computer science 3", "CSCI3110");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java]: CSCI2110 and capital S in Science

File: app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        db.course.insert("Server Side Scripting","INFX2670");
+        db.course.insert("Machine Learning", "CSCI4001");
+        db.course.insert("Designing User Interfaces","CSCI3640");
+        db.course.insert("Mobile Computing","CSCI4110");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java]: CSCI4176

File: app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        db.course.insert("Robotics","CSCI1400");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java]: Animated Computing, CSCI1106

File: app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        db.course.insert("Social Computing","CSCI3600");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java]: CSCI1107

File: app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ * Created by greg on 07/03/17.
+ */
+
+public class CourseSeeder {
+
+    public static void insert(DBHelper db){
+        db.course.insert("Computer science 3", "CSCI3110");
+        db.course.insert("Server Side Scripting","INFX2670");
+        db.course.insert("Machine Learning", "CSCI4001");
+        db.course.insert("Designing User Interfaces","CSCI3640");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java]: CSCI3160

File: app/src/main/java/cs/dal/krush/seeders/DatabaseSeeder.java [hunk 6]
```diff
@@ -35,17 +35,24 @@ public void insertData() {
...
+        LocationSeeder.insert(mydb);
+        SchoolSeeder.insert(mydb);
+        TutorSeeder.insert(mydb);
+        StudentSeeder.insert(mydb);
+        TutoringSessionSeeder.insert(mydb);
+        CourseSeeder.insert(mydb);
+        AudioRecordingSeeder.insert(mydb);
+        CoursesTutorsSeeder.insert(mydb);
+
+    } //end method
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/DatabaseSeeder.java]: Remove comment

File: app/src/main/java/cs/dal/krush/seeders/DatabaseSeeder.java [hunk 7]
```diff
@@ -91,5 +98,6 @@ public void insertData() {
         if (!rs.isClosed()) {
             rs.close();
         }
-    } //end method
+    }
+
 } // end class
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/DatabaseSeeder.java]: Remove comment

File: app/src/main/java/cs/dal/krush/seeders/LocationSeeder.java [hunk 8]
```diff
@@ -0,0 +1,20 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/LocationSeeder.java]: Change this comment to a description of the class

File: app/src/main/java/cs/dal/krush/seeders/SchoolSeeder.java [hunk 9]
```diff
@@ -0,0 +1,21 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/SchoolSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/CoursesTutorsSeeder.java [hunk 10]
```diff
@@ -0,0 +1,30 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CoursesTutorsSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java [hunk 11]
```diff
@@ -0,0 +1,20 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/CourseSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/AudioRecordingSeeder.java [hunk 12]
```diff
@@ -0,0 +1,25 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/AudioRecordingSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/DatabaseSeeder.java [hunk 13]
```diff
@@ -9,7 +9,7 @@
  * Created by greg on 27/02/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/DatabaseSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/StudentSeeder.java [hunk 14]
```diff
@@ -0,0 +1,21 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/StudentSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/StudentSeeder.java [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+ */
+
+public class StudentSeeder {
+
+    public static void insert(DBHelper db){
+        db.student.insert(1,"Greg","Miller","gregpmillr@gmail.com","password");
+        db.student.insert(3,"John","Wick","jw@gmail.com","password");
+        db.student.insert(2,"John","Smith","js@gmail.com","password");
+        db.student.insert(4,"Samuel","Jackson","sj@dal.ca","password");
+        db.student.insert(5,"Jack","Daniels","jd@outlook.com","password");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/StudentSeeder.java]: We should also add some girl names (or gender neutral names

File: app/src/main/java/cs/dal/krush/seeders/TutorSeeder.java [hunk 16]
```diff
@@ -0,0 +1,22 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/TutorSeeder.java]: Change to a description

File: app/src/main/java/cs/dal/krush/seeders/TutorSeeder.java [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+
+public class TutorSeeder {
+
+    public static void insert(DBHelper db){
+        db.tutor.insert(1,1,"Greg","Miller","gregpmillr@gmail.com","password",2);
+        db.tutor.insert(3,2,"Michael","Casey","mc@dal.ca","password",2);
+        db.tutor.insert(2,1,"Orjan","Monsen","om@dal.ca","password",5);
+        db.tutor.insert(4,5,"Eric","Desjardins","ed@dal.ca","password",1);
+        db.tutor.insert(5,3,"Connor","Walsh","cw@dal.ca","password",4);
+        db.tutor.insert(2,2,"Jack","Reaper","jr@dal.ca","password",2);
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/TutorSeeder.java]: Same thing here, might want to add some girls / gender neutral names

File: app/src/main/java/cs/dal/krush/seeders/TutoringSessionSeeder.java [hunk 18]
```diff
@@ -0,0 +1,20 @@
+package cs.dal.krush.seeders;
+
+import cs.dal.krush.models.DBHelper;
+
+/**
+ * Created by greg on 07/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/TutoringSessionSeeder.java]: Change to a class description

**Pull request merged**